static/js/app.js

```diff
@@ -34,7 +41,7 @@
     editor.on('change', function () {
       if ($('#inline-modal').is (':visible')) $('#inline-modal').hide();
       window.State.disable_run = false;
-      $ ('#runit').css('background-color', '');
+      //$ ('#runit').css('background-color', ''); Delete to prevent changing color within time-out
```

Comment:
This is being done now from inside the timeout, right? If so, could we just remove the line from here? As long as the default color of the button is shown when the user enters text, we should be alright.

Reply:
Yes I think this line can be deleted!